package.json

```diff
@@ -20,7 +20,7 @@
     "lint-fix": "eslint --fix . --ext .ts,.tsx",
     "build": "tsc",
     "watch": "tsc -w",
-    "test": "jest --silent --passWithNoTests --testPathPattern=src/*",
+    "test": "jest --silent --passWithNoTests --testPathPattern=src/* --runInBand",
```

Comment:
multitenancy tests run export jobs. Running in parallel sometimes hits the per user limit of concurrent export requests, so we run them serially.